v2/internal/config/vars.go

```diff
@@ -96,9 +100,9 @@ func (v Values) Cloud() cloud.Configuration {
 
 	// Special handling if we've got all the defaults just return the official public cloud
 	// configuration
-	hasDefaultAzureAuthorityHost := v.AzureAuthorityHost == "" || v.AzureAuthorityHost == cloud.AzurePublic.ActiveDirectoryAuthorityHost
-	hasDefaultResourceManagerEndpoint := v.ResourceManagerEndpoint == "" || v.ResourceManagerEndpoint == cloud.AzurePublic.Services[cloud.ResourceManager].Endpoint
-	hasDefaultResourceManagerAudience := v.ResourceManagerAudience == "" || v.ResourceManagerAudience == cloud.AzurePublic.Services[cloud.ResourceManager].Audience
+	hasDefaultAzureAuthorityHost := v.AzureAuthorityHost == "" || v.AzureAuthorityHost == DefaultAADAuthorityHost
```

Comment:
You have similar changes in your other [PR](https://github.com/Azure/azure-service-operator/pull/2445) which might be conflicting

Reply:
Fixed